### ISSUE TITLE
@bind text >32kb

### ISSUE THREAD

**Issue by cores-system** [aspnet/AspNetCore#11184]
Hello, there is a simple form

```
<textarea @bind="val"></textarea>

@code{ 
string val;
}
```

If a user tries to insert text more than 32kb into a form, he gets an error: `Error: Connection disconnected with error 'Error: Server returned an error on close: Connection closed with an error.'.`

Is there a workaround for getting custom text over 32kb ?
I am using Blazor Preview 6

**Comment by cores-system** [comment]
I tried to change the size of the buffer

```
app.UseRouting();
app.UseEndpoints(endpoints =>
{
	endpoints.MapDefaultControllerRoute();
	endpoints.MapBlazorHub(o => 
	{
		o.TransportMaxBufferSize = 1000000;
		o.ApplicationMaxBufferSize = 1000000;
	});
	endpoints.MapFallbackToPage("/_Host");
});
```

But it does not work, the text size is still limited to 32kb.

**Comment by davidfowl** [comment]
No that does not work, you need to do this now:

```C#
services.AddServerSideBlazor().AddHubOptions(o =>
            {
                o.MaximumReceiveMessageSize = ...;
            });
```

**Comment by QadiymStewart** [comment]
I can comfirm that works now fixed my issue #9570

**Comment by cores-system** [comment]
Thanks, it really works 👍
